packages/components/src/Header.tsx

```diff
@@ -197,7 +197,7 @@ export const Header = ({
                             display: 'inline-block',
                         }}
                     >
-                        {titleArray[1] && titleArray[0]}
+                        {titleArray[1] && titleArray[0]}/
```

Comment:
There is an issue with this if the title is just one word. `/` shouldn't stay if the title is just one word.
You can see it here
https://60d9b2ac2cf86a00396f9b0f-lgjetjlwos.chromatic.com/?path=/story/components-header--disconnected

Reply:
Could you check it now?

Reply:
It's working, good solution!

Reply:
Thanks 👍